cumulus/parachains/runtimes/bridge-hubs/bridge-hub-rococo/src/bridge_to_bulletin_config.rs

```diff
@@ -0,0 +1,267 @@
+	/// unconfirmed messages that the single confirmation transaction at Rococo Bulletin Chain may process.
+	pub const MaxUnconfirmedMessagesAtInboundLane: bp_messages::MessageNonce =
+		bp_polkadot_bulletin::MAX_UNCONFIRMED_MESSAGES_IN_CONFIRMATION_TX;
+	/// Bridge specific chain (network) identifier of the Rococo Bulletin Chain.
+	pub const RococoBulletinChainId: bp_runtime::ChainId = bp_runtime::POLKADOT_BULLETIN_CHAIN_ID;
+	/// Interior location (relative to this runtime) of the with-RococoBulletin messages pallet.
+	pub BridgeRococoToRococoBulletinMessagesPalletInstance: InteriorMultiLocation = X1(
+		PalletInstance(<BridgeRococoBulletinMessages as PalletInfoAccess>::index() as u8)
+	);
+	/// Rococo Bulletin Network identifier.
+	pub RococoBulletinGlobalConsensusNetwork: NetworkId = NetworkId::PolkadotBulletin;
+	/// All active lanes that the current bridge supports.
+	pub ActiveOutboundLanesToRococoBulletin: &'static [bp_messages::LaneId]
+		= &[XCM_LANE_FOR_ROCOCO_PEOPLE_TO_ROCOCO_BULLETIN];
+	/// Lane identifier, used to connect Rococo People and Rococo Bulletin chain.
+	pub const RococoPeopleToRococoBulletinMessagesLane: bp_messages::LaneId
+		= XCM_LANE_FOR_ROCOCO_PEOPLE_TO_ROCOCO_BULLETIN;
+	/// Priority boost that the registered relayer receives for every additional message in the message
+	/// delivery transaction.
+	///
+	/// It is determined semi-automatically - see `FEE_BOOST_PER_MESSAGE` constant to get the
+	/// meaning of this value.
+	pub PriorityBoostPerMessage: u64 = 182_044_444_444_444;
+
+	/// Identifier of the sibling Rococo People parachain.
+	pub RococoPeopleParaId: cumulus_primitives_core::ParaId = 1004.into();
+	/// A route (XCM location and bridge lane) that the Rococo People Chain -> Rococo Bulletin Chain
+	/// message is following.
+	pub FromRococoPeopleToRococoBulletinRoute: SenderAndLane = SenderAndLane::new(
+		ParentThen(X1(Parachain(RococoPeopleParaId::get().into()))).into(),
+		XCM_LANE_FOR_ROCOCO_PEOPLE_TO_ROCOCO_BULLETIN,
+	);
+
+	/// XCM message that is never sent.
+	pub NeverSentMessage: Option<Xcm<()>> = None;
+}
+pub const XCM_LANE_FOR_ROCOCO_PEOPLE_TO_ROCOCO_BULLETIN: LaneId = LaneId([0, 0, 0, 0]);
+
+/// Proof of messages, coming from Rococo Bulletin chain.
+pub type FromRococoBulletinMessagesProof =
+	FromBridgedChainMessagesProof<bp_polkadot_bulletin::Hash>;
+/// Messages delivery proof for Rococo Bridge Hub -> Rococo Bulletin messages.
+pub type ToRococoBulletinMessagesDeliveryProof =
+	FromBridgedChainMessagesDeliveryProof<bp_polkadot_bulletin::Hash>;
+
+/// Dispatches received XCM messages from other bridge.
+type FromRococoBulletinMessageBlobDispatcher = BridgeBlobDispatcher<
+	XcmRouter,
+	BridgeHubRococoUniversalLocation,
+	BridgeRococoToRococoBulletinMessagesPalletInstance,
+>;
+
+/// Export XCM messages to be relayed to the other side.
+pub type ToRococoBulletinHaulBlobExporter = HaulBlobExporter<
```

Comment:
I think we could use `pallet_xcm_bridge_hub` here also like for to_westend stuff: https://github.com/paritytech/polkadot-sdk/blob/master/cumulus/parachains/runtimes/bridge-hubs/bridge-hub-rococo/src/bridge_to_westend_config.rs#L129C47-L129C70

Reply:
Yep, added the other instance of BH pallet